src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -183,13 +183,12 @@ export default class BrowseSkill extends React.Component {
 
   handleNavigationBackward = () => {
     $('html, body').animate({ scrollTop: 0 }, 'fast');
-    const { listPage, listOffset, entriesPerPage, skills } = this.state;
```

Comment:
Why was the destructuring removed? Please use destructuring.

Also, you can use the following syntax here - 
```
this.setState( (prevState) => { listPage : prevState.listPage - 1 } );

```

Reply:
@akshatnitd Done the changes.